### ISSUE TITLE
Resolve error: unable to load resolver "node".

### ISSUE THREAD

**Issue by sebald** [azz/eslint-plugin-monorepo#8]
If `eslint-plugin-import` is not installed the plugin will throw the following error:

```
Resolve error: unable to load resolver "node".  monorepo/no-relative-import
```

when trying to resolve an `import` statement.

I not sure why installing `eslint-plugin-import` solves this issue (maybe incompatible with `@typescript-eslint`?). Is there a better way to fix this?

**Comment by azz** [comment]
Would you be able to make a minimal repo to reproduce this issue?

**Comment by sebald** [comment]
Of course! Here you go: https://github.com/sebald/eslint-plugin-monorepo-typescript

I noticed that I haven't mentioned that we are using `@typescript-eslint`. Sorry about that ☹️ I guess this is an incompatibility with the plugin?

**Comment by ghost** [comment]
Yeah i met the exactly same issue. Using `@typescript-eslint` as well.

**Comment by kwelch** [comment]
I can confirm this worked properly after adding in the `eslint-plugin-import`. We had not configured that plugin, but we will use it eventually, so not terrible, but slightly unfortunate.

**Comment by constgen** [comment]
This may be related to this bug which despite the status is not actually fixed https://github.com/benmosher/eslint-plugin-import/issues/1642 . As I discovered this happens not because of Typescript but becuase is of Yarn 2 and `eslint-plugin-import` doesn consider that it may be reused in another 3rd party config

**Comment by electrovir** [comment]
For me, installing `eslint-import-resolver-node` alone solved this issue.
